packages/eslint-plugin-pf-codemods/src/ruleCustomization.ts

```diff
@@ -69,6 +69,7 @@ export const setupRules = [
 
 // rules that will run after other rules (cleanup imports?)
 export const cleanupRules = [
+  "data-codemods-cleanup",
```

Comment:
This will run the rule after every pass of a consumer running codemod fixes correct?

Wondering if we would want that to be the case. Using the v5 codemods that updated TableComposable to just Table and the then-currentl implementation of Table to TableDeprecated, if we removed a data-codemod attr from the TableComposable fixer, I imagine it would cause a scenario of TableComposable > updated to Table > updated to TableDeprecated incorrectly.

If that's the case, I feel like this rule might be better off as one that a consumer has to run explicitly, with us stating that they should only run it when they are finished running our codemods.

Reply:
Yeah, you are right. I just tested that and on the first run of fixes it does everything correctly, but when you run the general codemod fixes again, it will show unwanted errors and do the unwanted fixes, due to the missing `data-codemods`. Which could lead to a lot of trouble, so yes, I'll change that rule to be runnable with only the `--only` option. Also it is working as v6 only rule right now, so that must change too.